braph2genesis/measures/_Distance.gen.m

```diff
@@ -945,3 +945,204 @@
 assert(isequal(m_inside_g.get('M'), known_distance), ...
     [BRAPH2.STR ':Distance:' BRAPH2.FAIL_TEST], ...
     [class(m_inside_g) ' is not being calculated correctly for ' class(g) '.'])
+
+%%% ¡test!
+%%%% ¡name!
+OrdMxBUD
+%%%% ¡probability!
+.01
+%%%% ¡code!
+B = [
+    0   .2   .7
+    .2   0   .1
+    .7  .1   0
+    ];
+
+densities = [0 33 67 100];
+
+known_distance = {
+    [
+    0   Inf Inf
+    Inf 0   Inf
+    Inf Inf 0    
+    ]
+    [
+    0   Inf Inf
+    Inf 0   Inf
+    Inf Inf 0    
+    ]
+    [
+    0   Inf Inf
+    Inf 0   Inf
+    Inf Inf 0    
+    ]
+    [
+    0   Inf 1
+    Inf 0   Inf
+    1   Inf 0
+    ]
+    [
+    0   Inf 1
+    Inf 0   Inf
+    1   Inf 0
+    ]
+    [
+    0   Inf 1
+    Inf 0   Inf
+    1   Inf 0
+    ]
+    [
+    0   1   1
+    1   0   2
+    1   2   0
+    ]
+    [
+    0   1   1
+    1   0   2
+    1   2   0
+    ]
+    [
+    0   1   1
+    1   0   2
+    1   2   0
+    ]
+    [
+    0   1   1
+    1   0   1
+    1   1   0
+    ]
+    [
+    0   1   1
+    1   0   1
+    1   1   0
+    ]
+    [
+    0   1   1
+    1   0   1
+    1   1   0
+    ]
+    };
+
+g = OrdMxBUD('B', {B B B}, 'DENSITIES', densities);
+
+m_outside_g = Distance('G', g);
+assert(isequal(m_outside_g.get('M'), known_distance), ...
+    [BRAPH2.STR ':Distance:' BRAPH2.FAIL_TEST], ...
+    [class(m_outside_g) ' is not being calculated correctly for ' class(g) '.'])
+
+m_inside_g = g.get('MEASURE', 'Distance');
+assert(isequal(m_inside_g.get('M'), known_distance), ...
+    [BRAPH2.STR ':Distance:' BRAPH2.FAIL_TEST], ...
+    [class(m_inside_g) ' is not being calculated correctly for ' class(g) '.'])
+
+%%% ¡test!
+%%%% ¡name!
+MultilayerWU
+%%%% ¡probability!
+.01
+%%%% ¡code!
+B11 = [
+    0       .1  .2  .25  0  0;
+    .125    0   0   0    0  0;
+    .2      .5  0   .25  0  0;
+    .125    10  0   0    0  0;
+    0       0   0   0    0  0;
+    0       0   0   0    0  0 
+    ];
+B22 = [
+    0       .1  .2  .25  0;
+    .125    0   0   0    0;
+    .2      .5  0   .25  0;
+    .125    10  0   0    0;
+    0       0   0   0    0;
+    ];
+B12 = rand(size(B11,1),size(B22,2));
```

Comment:
Leave a space after comman, i.e. 
"B12 = rand(size(B11, 1), size(B22, 2));"
And distribute the changes across the script and keep the style consistent